examples/basic-cli/src/main.rs

```diff
@@ -43,46 +54,142 @@ fn main() {
     // Start reading data
     println!("Opened uBlox device, waiting for messages...");
     loop {
-        device
-            .on_data_available(|packet| match packet {
-                PacketRef::MonVer(packet) => {
-                    println!(
-                        "SW version: {} HW version: {}; Extensions: {:?}",
-                        packet.software_version(),
-                        packet.hardware_version(),
-                        packet.extension().collect::<Vec<&str>>()
-                    );
-                    println!("{packet:?}");
-                },
-                PacketRef::NavPvt(pvt) => {
-                    let has_time = pvt.fix_type() == GnssFixType::Fix3D
-                        || pvt.fix_type() == GnssFixType::GPSPlusDeadReckoning
-                        || pvt.fix_type() == GnssFixType::TimeOnlyFix;
-                    let has_posvel = pvt.fix_type() == GnssFixType::Fix3D
-                        || pvt.fix_type() == GnssFixType::GPSPlusDeadReckoning;
-
-                    if has_posvel {
-                        let pos: Position = (&pvt).into();
-                        let vel: Velocity = (&pvt).into();
-                        println!(
-                            "NavPvt: Latitude: {:.5} Longitude: {:.5} Altitude: {:.2} m, Speed: {:.2} m/s Heading: {:.2} degrees",
-                            pos.lat, pos.lon, pos.alt
-                            ,vel.speed, vel.heading
-                        );
-                        println!("NavPvt full: {pvt:?}");
+        device.on_data_available(|packet| {
```

Comment:
Wouldn't it be better to create 3 separate examples? What do you think?

Reply:
Yes. Can we handle it in a following PR? Maybe we can also make a more minimal example. We don't have any simple examples.